packages/litexa/src/parser/skill.coffee

```diff
@@ -293,12 +292,12 @@ class lib.Skill
       @sayMapping[location.language] = [{ from, to }]
 
   pushDBTypeDefinition: (definition) ->
```

Comment:
Could you please update the integration test for `db-classes` or `localization` to cover DB type localization and fallback to default? Should fail pre-change and succeed post-change.